src/amo/components/CollectionManager/index.js

```diff
@@ -80,10 +80,11 @@ export class CollectionManagerBase extends React.Component<
   }
 
   onCancel = (event: SyntheticEvent<HTMLButtonElement>) => {
-    const { creating, dispatch, history } = this.props;
+    const { clientApp, creating, dispatch, history, siteLang } = this.props;
 
     if (creating) {
-      history.goBack();
+      // This is used over history.goBack in case the language changes.
```

Comment:
I don't think we need this comment, as we can no longer see that this used to call `history.goBack`, so I'm not sure what the comment adds.

Reply:
ok yeh I guess it loses context so yep I'll remove.. thanks : )